### PR TITLE
Feat/netbox 4 1

### PR DESCRIPTION
**Pull request by cruse1977**
Netbox v4.1 support PR

### CODE REVIEW

File: netbox_bgp/__init__.py [hunk 1]
```diff
@@ -11,8 +11,8 @@ class BGPConfig(PluginConfig):
     author_email = 'mgk.kolek@gmail.com'
     base_url = 'bgp'
     required_settings = []
-    min_version = '4.0.0'
-    max_version = '4.0.99'
+    min_version = '4.1.0'
+    max_version = '4.1.20'
```
Comment (berendt) [inline netbox_bgp/__init__.py:15]: Wouldn't it be better to use 4.1.99?

**Comment by aascedu** [comment]
Could someone merge this PR so I can start using netbox-bgp please ❤️

**Comment by kevinsdirectory** [comment]
Really want to use 4.1 with bgp plugin, can you push this forward please. Much appreciated for all the hard work you do!

**Comment by aascedu** [comment]
> Really want to use 4.1 with bgp plugin, can you push this forward please. Much appreciated for all the hard work you do!

Hello, if you really need to have netbox-bgp in your netbox, you can either git clone the forked repo I have on my github profil, or fork that repo and change the file in netbox-bgp/__init__.py to allow up to 4.1.99 version.
Then you can do `/opt/netbox/venv/bin/pip install .` at the root of the git repo.

I installed it for my team this morning and everything seems to work fine ;) Don't forget to add it to your netbox plugins in the config file (/etc/netbox/config/plugins.py
If you use netbox-docker, the process is a bit longer !

Have a good day !
Arthur

**Comment by kevinsdirectory** [comment]
@aascedu I am currently using netbox-docker and the process is bit more complicating than it needs to be for a minor change. I am just wondering when this update might happen?

**Comment by aascedu** [comment]
@kevinsdirectory here is what you need to change in the Dockerfile-Plugins
to be able to install from a local repo...
```
RUN /opt/netbox/venv/bin/pip install  --no-warn-script-location -r
/opt/netbox/plugin_requirements.txt && \
  apt-get update -y && \
  apt-get install -y git && \
  git clone https://github.com/aascedu/netbox-bgp.git && \
  /opt/netbox/venv/bin/pip install --no-warn-script-location ./netbox-bgp
```
I don't think it is that bad... As you want, you can keep begging or get
working !

**Pull request closed without merging**